styles.css

```diff
@@ -0,0 +1,36 @@
+body {
+  font-family: Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
+}
+
+.nav-bar {
+  display: flex;
+  justify-content: space-around;
+  padding-left: 30px;
+  flex: 1;
+}
```

Comment:
- [ ] The color is different from the main template, and also the  the position of them are really far.
![nnnbbb](https://user-images.githubusercontent.com/61562626/129912779-b6fc6ae6-5d0e-45aa-8c98-61a8bf6f588c.jpeg)

- And your 
![,,,,,,](https://user-images.githubusercontent.com/61562626/129914002-5cf2f521-7dc3-4d66-8d12-754fe2312908.jpeg)